doc/development/glosary.md

```diff
@@ -0,0 +1,28 @@
+# Puter's glosary
+
+## General
+
+1. **Puter:** Puter is an advanced, open-source internet operating system designed to be feature-rich, exceptionally fast, and highly extensible.
+
+2. **Self hosting:** Means running and managing the app on your infrastructure instead of using thid-party hosting service.
```

Comment:
"Puter" is more clear than "the app" because Puter also runs apps.

```suggestion
2. **Self hosting:** Means running and managing Puter on your infrastructure instead of using thid-party hosting service.
```